### DISCUSSION TITLE
Add proper search via something like meilisearch

### DISCUSSION THREAD

**Discussion by mysticaltech** [danny-avila/LibreChat#51] [Feature Requests & Suggestions]
Hey Daniel, I know you plan on supporting proper search of the conversations, I just wanted to remind you of meilisearch, one of the best and easiest open-source search solutions, it can work on docker too! https://github.com/meilisearch/meilisearch

**Comment by mysticaltech** [comment]
Also, it will allow you to search the whole conversation, which would be much better than just the title. Just have a look at meili, it may be a lot easier to integrate than you imagine!

**Comment by danny-avila** [comment]
> Also, it will allow you to search the whole conversation, which would be much better than just the title. Just have a look at meili, it may be a lot easier to integrate than you imagine!

I will for sure take a look, I'm curious how it works!

**Comment by danny-avila** [comment]
This is going to be next on my list btw.

This would be a nice solution if it still works (mongodb sync for meilisearch, project is archived though): https://github.com/Loophole-Labs-Archive/mongomeili

I could start indexing with meili without any library, but I'm curious how much overhead that would add.

**Comment by mysticaltech** [comment]
@danny-avila Sounds good. If I were you, try instead using the [meilisearch-js](https://github.com/meilisearch/meilisearch-js) lib, to create and then update the index whenever a document changes. Seems super easy!

**Comment by danny-avila** [comment]
@mysticaltech Just wanted to let you know this is finally live with the latest release! 

https://github.com/danny-avila/chatgpt-clone/releases/tag/v0.1.0

**Reply by mysticaltech** [reply]
Well done @danny-avila 🥳 🙏
